packages/libglib/Cargo.toml

```diff
@@ -12,8 +12,8 @@ path = "../packages.rs"
 releases-url = "https://download.gnome.org/sources/glib"
 
 [[package.metadata.build-package.external-files]]
-url = "https://download.gnome.org/sources/glib/2.77/glib-2.77.2.tar.xz"
-sha512 = "0e2cf5178a7174b174480a795b1cbb2f1dbbea35899b7e4756937e426b6c39c20deb8488958403b78552674c4179e6fd63ea7fac2b746ac49a62046e27086111"
+url = "https://download.gnome.org/sources/glib/2.77/glib-2.77.3.tar.xz"
```

Comment:
There is a slightly newer version of `glib` if we want to take that as well: https://download.gnome.org/sources/glib/2.78/

Reply:
Updated to latest.